Userland/Libraries/LibJS/Runtime/Temporal/Calendar.cpp

```diff
@@ -192,4 +301,119 @@ i32 iso_days_in_month(i32 year, i32 month)
     return 28;
 }
 
+// 12.1.36 BuildISOMonthCode ( month )
+String build_iso_month_code(i32 month)
+{
+    return String::formatted("M{:02}", month);
+}
+
+// 12.1.37 ResolveISOMonth ( fields ), https://tc39.es/proposal-temporal/#sec-temporal-resolveisomonth
+double resolve_iso_month(GlobalObject& global_object, Object& fields)
```

Comment:
do you think it makes sense to return an `u8` here instead?

Reply:
Nope, if monthCode is undefined and month isn't we just return month, which can be any arbitrary double.